src/libs/actions/TransactionEdit.ts

```diff
@@ -27,10 +33,10 @@ function removeBackupTransaction(transactionID: string) {
 }
 
 function restoreOriginalTransactionFromBackup(transactionID: string, isDraft: boolean) {
-    const connection = Onyx.connect({
+    connectionID = Onyx.connect({
         key: `${ONYXKEYS.COLLECTION.TRANSACTION_BACKUP}${transactionID}`,
         callback: (backupTransaction) => {
-            Onyx.disconnect(connection);
+            Onyx.disconnect(connectionID);
```

Comment:
The new Onyx version `disconnect` function 's param now requires the `Connection` type, why do we have this change?

```javascript
 * @param connection Connection object returned by calling `Onyx.connect()`.
 */
declare function disconnect(connection: Connection): void;
/**
```

Reply:
Oh, I haven't installed the new onyx. Updated.